go.mod

```diff
@@ -33,13 +33,14 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.12.0
 	github.com/stretchr/testify v1.8.1
-	github.com/tinkerbell/rufio v0.0.0-20220606134123-599b7401b5cc
-	github.com/tinkerbell/tink v0.7.1-0.20221004171112-6deeea887dac
+	github.com/tinkerbell/cluster-api-provider-tinkerbell v0.4.0
```

Comment:
I am not sure we want to update these tags. There is like a whole initiative to upgrade tinkerbell dependencies and the stack itself. There were some complications due to API changes.

Reply:
This change was probably added because of importing CAPT to AddToScheme for tinkerbell.

Reply:
The changes are introduced by importing CAPT, but I required rufio revision at the bottom

Reply:
But there is a breaking api change in rufio that will cause other issues. Can we just import an older version of CAPT instead? That will simplify things here.

Reply:
That's why I require an older version of rufio at [bottom](https://github.com/aws/eks-anywhere/pull/4764/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R214). 
We didn't have CAPT imported, is there a specific version in your mind, or the latest that doesn't change rufio?

Reply:
If we want to have minimum impact we might want to use the same commit we build CAPT on.

https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/tinkerbell/cluster-api-provider-tinkerbell/GIT_TAG